### PR TITLE
Fixed label logic

### PR DESCRIPTION
**Pull request by bodya4243**
1) change create method in controller/service
2) change delete method in service

### CODE REVIEW

File: src/main/java/app/controller/LabelController.java [hunk 1]
```diff
@@ -26,11 +26,12 @@
 public class LabelController {
     private final LabelService labelService;
 
-    @PostMapping
+    @PostMapping("/{taskId}")
```
Comment (makaroshyna) [inline src/main/java/app/controller/LabelController.java:29]: It is bad practice to add ID of another entity here. It is best to use RequestParam anotation before taskId and remove this.

File: src/main/java/app/controller/LabelController.java [hunk 2]
```diff
@@ -26,11 +26,12 @@
...
     @ResponseStatus(HttpStatus.CREATED)
     @Operation(summary = "Label creation", description = "Create and save a new label")
-    public LabelResponseDto createLabel(@Valid @RequestBody LabelRequestDto dto) {
-        return labelService.create(dto);
+    public LabelResponseDto createLabel(@Valid @RequestBody LabelRequestDto dto,
+                                        @PathVariable Long taskId) {
```
Comment (makaroshyna) [inline src/main/java/app/controller/LabelController.java:33]:
```suggestion
                                        @RequestParam Long taskId) {
```

File: src/main/java/app/controller/LabelController.java [hunk 3]
```diff
@@ -26,11 +26,12 @@
...
+        return labelService.create(dto, taskId);
     }
 
     @GetMapping("/by_project/{projectId}")
```
Comment (makaroshyna) [inline src/main/java/app/controller/LabelController.java:37]: Here too. Let's replace PathVariable with RequestParam and remove this.
```suggestion
    @GetMapping
```

**Review by makaroshyna — CHANGES_REQUESTED** [review]
Just some minor comments about conroller methods. It will be best to change it.

**Review by makaroshyna — APPROVED** [review]

**Review by VlasenkoYevhenii — APPROVED** [review]

**Pull request merged**